modules/ztp-checking-du-cluster-config.adoc

```diff
@@ -0,0 +1,577 @@
+        pciAddress: 0000:3b:0a.3
+        vendor: "8086"
+        vfID: 3
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.4
+        vendor: "8086"
+        vfID: 4
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.5
+        vendor: "8086"
+        vfID: 5
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.6
+        vendor: "8086"
+        vfID: 6
+      - deviceID: 154c
+        driver: vfio-pci
+        pciAddress: 0000:3b:0a.7
+        vendor: "8086"
+        vfID: 7
+----
+
+. Check that the cluster performance profile is correct. The `cpu` and `hugepages` sections will vary depending on your hardware configuration. Run the following command:
++
+[source,terminal]
+----
+$ oc get PerformanceProfile openshift-node-performance-profile -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  creationTimestamp: "2022-07-19T21:51:31Z"
+  finalizers:
+  - foreground-deletion
+  generation: 1
+  name: openshift-node-performance-profile
+  resourceVersion: "33558"
+  uid: 217958c0-9122-4c62-9d4d-fdc27c31118c
+spec:
+  additionalKernelArgs:
+  - idle=poll
+  - rcupdate.rcu_normal_after_boot=0
+  - efi=runtime
+  cpu:
+    isolated: 2-51,54-103
+    reserved: 0-1,52-53
+  hugepages:
+    defaultHugepagesSize: 1G
+    pages:
+    - count: 32
+      size: 1G
+  machineConfigPoolSelector:
+    pools.operator.machineconfiguration.openshift.io/master: ""
+  net:
+    userLevelNetworking: true
+  nodeSelector:
+    node-role.kubernetes.io/master: ""
+  numa:
+    topologyPolicy: restricted
+  realTimeKernel:
+    enabled: true
+status:
+  conditions:
+  - lastHeartbeatTime: "2022-07-19T21:51:31Z"
+    lastTransitionTime: "2022-07-19T21:51:31Z"
+    status: "True"
+    type: Available
+  - lastHeartbeatTime: "2022-07-19T21:51:31Z"
+    lastTransitionTime: "2022-07-19T21:51:31Z"
+    status: "True"
+    type: Upgradeable
+  - lastHeartbeatTime: "2022-07-19T21:51:31Z"
+    lastTransitionTime: "2022-07-19T21:51:31Z"
+    status: "False"
+    type: Progressing
+  - lastHeartbeatTime: "2022-07-19T21:51:31Z"
+    lastTransitionTime: "2022-07-19T21:51:31Z"
+    status: "False"
+    type: Degraded
+  runtimeClass: performance-openshift-node-performance-profile
+  tuned: openshift-cluster-node-tuning-operator/openshift-node-performance-openshift-node-performance-profile
+----
++
+[NOTE]
+====
+CPU settings are dependent on the number of cores available on the server and should align with workload partitioning settings. `hugepages` configuration is server and application dependent.
+====
+
+. Check that the `PerformanceProfile` was successfully applied to the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get performanceprofile openshift-node-performance-profile -o jsonpath="{range .status.conditions[*]}{ @.type }{' -- '}{@.status}{'\n'}{end}"
```

Comment:
You need to [escape](https://docs.asciidoctor.org/asciidoc/latest/subs/prevent/) the asterisk or it'll make the portal build fail.

Reply:
Really? even inside a codeblock? from 

> By default, source blocks (as well as other verbatim blocks) are only subject to the verbatim substitution group (specialchars and callouts).

from: https://docs.asciidoctor.org/asciidoc/latest/subs/apply-subs-to-blocks/#set-the-subs-attribute-on-a-block

Reply:
Build passes portal :)